### ISSUE TITLE
Some objects are not synced

### ISSUE THREAD

**Issue by ameoba32** [amark/gun#1295]
I am facing weird issue, some objects are synced but some not. The only difference is object property name. Here is an example:

First textarea syncs, second one does not. Can you help?

https://jsfiddle.net/gvfod47x/24/

```html
<!DOCTYPE html>
<textarea id="paste1" placeholder="Test 1 works"></textarea>
<textarea id="paste2" placeholder="Test 2 does not work"></textarea>
<script src="https://cdn.jsdelivr.net/npm/gun/gun.js"></script>
<script>
  gun = Gun(['https://gun-manhattan.herokuapp.com/gun']);
  // This one works
  let copy1 = gun.get('test1');
  paste1.oninput = () => { copy1.put({serving:paste1.value, test:'test'}) };
  copy1.on((data) => { paste1.value = data.serving });
  
  // This one does not work, only difference is "test" => "mest"
  let copy2 = gun.get('test2');
  paste2.oninput = () => { copy2.put({serving:paste2.value, mest:'test'}) };
  copy2.on((data) => { paste2.value = data.serving });
</script>
```

**Comment by ggcaponetto** [comment]
Banged my head on this a couple of hours too. :(

**Comment by ameoba32** [comment]
I've tried to go deeper. Seems it depends on the order object properties are sent to gun server.  In these two examples order is different.

Then I've tried to debug the server... omg. Code looks like it is obfuscated, but it is not, it is like this. Very hard to read, impossible to understand logic, full of hacks. 

I see no future of it, I gave up. Once I understand I spend more time on gun library than on project itself, I switched to another library :(

**Comment by amark** [comment]
@ggcaponetto brought this too my attention again and I was able to confirm/replicate. **This has moved to highest priority**. Last time with Jonah it only happened with svelte (we couldn't isolate) so it dropped off my mind, but I'm glad ggcaponetto pestered me (I've been mostly focused on AXE, RAD, and SecureRender). 

Why did you close the issue? Please don't do that. I'm just a guy, who is very stressed, i get so many robot notifications I miss a lot of things.

**Comment by amark** [comment]
Seems related to AXE, if you turn AXE off on relay `GUN({axe: false, ...` this bug goes away. Still need to investigate & fix.

**Comment by noctisatrae** [comment]
That's why it was so buggy during my hackathon project. This bug really is horrible to deal with. How can I help ?

**Comment by amark** [comment]
@noctisatrae how long ago was your hackathon? :/

I have a fix, but one of the PANIC tests isn't passing... tho, looks like this PANIC test didn't pass for the currently-affected version too. So maybe I should proceed anyways? Gah, always scared & trying to be cautious.

**Comment by noctisatrae** [comment]
I coded for 1 month, and there was this weird bug with AXE where things were not synced. 

So I'll try to help you because that's a major issue :D ! 

I'll try to look at the tests. What is the branch in which you PRed the fix ?

**Comment by amark** [comment]
@noctisatrae 👍 

So the failing PANIC test failed on the old version too.

I can publish now to fix the major issue

however, various AXE improvements still have some annoying edge cases: It'll prevent GUN from syncing if a new peer joins and there is uninitialized/empty data, AXE will only sync correctly after data already exists to be found.

Should I go ahead an publish the more important fix now, even tho it may cause other things to break?

**Comment by amark** [comment]
@ameoba32 @ggcaponetto @noctisatrae @i001962 fixed the AXE bug. Published in GUN v`0.2020.1239`.

PLEASE UPGRADE ALL RELAYS. 

easy to check what version a relay is running, just add: /gun/package.json to the URL

Spread the word. We need all relays to upgrade, since this was a bug in AXE, not GUN. Even if browsers are on latest GUN if they're connected to a deprecated `1238` relay with AXE turned on, then people will still hit the same bug.

Please spread the word. Get everyone to upgrade.

Thanks to @rogowski for helping me get this out the door before Christmas.
